test/integration/test_api_build_test_container.py

```diff
@@ -0,0 +1,58 @@
+from exasol_integration_test_docker_environment.lib import api
+from exasol_integration_test_docker_environment.lib.docker import ContextDockerClient
+from exasol_integration_test_docker_environment.lib.docker.images.image_info import (
+    ImageState,
+)
+from exasol_integration_test_docker_environment.test.get_test_container_content import (
+    MOCK_TEST_CONTAINER_PATH,
+    get_test_container_content,
+)
+
+TEST_CONTAINER_CONTENT = get_test_container_content(MOCK_TEST_CONTAINER_PATH)
+
+
+def test_build_test_container(api_isolation):
+    docker_repository_name = api_isolation.docker_repository_name
+    image_info = api.build_test_container(
+        target_docker_repository_name=docker_repository_name,
+        test_container_content=TEST_CONTAINER_CONTENT,
+        output_directory=api_isolation.output_dir,
+    )
+    with ContextDockerClient() as docker_client:
+        image = docker_client.images.get(image_info.get_target_complete_name())
+        assert len(image.tags) == 1
```

Comment:
Actually, not so great for debugging, the output you get out of these checks. When the image is not found, you don't see, which other images are there. Maybe, somewhere, you have a slight error in creating the name and when you see the list of images, that could help you to figure out what is going on.

Reply:
I don't agree. 
Looks at the following snippet:
```
tags = ["abc", "def", "ghi"]
```

Then pytests prints:
```
            tags = ["abc", "def", "ghi"]
>           assert len(tags) == 1
E           AssertionError: assert 3 == 1
E            +  where 3 = len(['abc', 'def', 'ghi'])
```

Reply:
Fixed